src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -1103,13 +1103,13 @@ public static String escapeXpathValue(String value) {
 		return String.format("'%s'", value);
 	}
 
-	private boolean textContains(String actual, String text, boolean ignoreCase) {
+	public static boolean textContains(String actual, String text, boolean ignoreCase) {
```

Comment:
I personally don't see a reason to declare these methods as public but it shouldn't hurt.

Reply:
without making them public, making them static doesn't really make a lot of sense, since they are never called from a static method within the class itself

Reply:
Yes, I agree that there is no "a lot of sense" and this topic is... debatable. From my point of view, "private static" is still make sense because
1) it emphasize the fact that this method doesn't depend on other class members (doesn't use internal state)
2) JVM doesn't pass `this` as the first parameter to such the methods

Here are my points. The last call, of course, in on @Hi-Fi and you :)

Reply:
I'm doing one more commit. Do I leave this in as public static, or revert it back to private?
private static doesn't make much sense since in the class it is never called from a static context.

Reply:
For me both are ok. Maybe original is better if there's no clear reason for change.